app/views/pages/home/_filters.html.haml

```diff
@@ -9,5 +9,5 @@
         = t("jobs.filters.#{filter_type}")
 
     .checkbox-group__container{ aria: { labelledby: "checkbox-group-heading-#{filter_type}" } }
-      = render 'shared/checkbox_group', search: false, f: f, attribute_name: filter_type.to_sym, options: @jobs_search_form.send("#{filter_type.chop}_options".to_sym), value_method: :first, text_method: :last, small: true
+      = render 'shared/checkbox_group', classes: '', search: false, f: f, attribute_name: filter_type.to_sym, options: @jobs_search_form.send("#{filter_type.chop}_options".to_sym), value_method: :first, text_method: :last, small: true
```

Comment:
I think you should be able to omit `classes: ''`, it should be ok for it to be nil

Reply:
classes has to exist according to error when it is omitted